Source/ExecutionEngine/ExecutionEngine.cs

```diff
@@ -1750,7 +1735,8 @@ private ErrorInformation CreateErrorInformation(Counterexample error, VC.VCGen.O
         {
           if (assertError.FailingAssert.ErrorMessage == null || Options.ForceBplErrors)
           {
-            string msg = assertError.FailingAssert.ErrorData as string ?? "This assertion might not hold.";
+            string msg = assertError.FailingAssert.ErrorData as string ??
```

Comment:
ErrorData and Description are both ways to customise the error message. Shouldn't there be only one?

Reply:
Marking it as `[Obsolete]` leads to a slew of compilation warnings, and probably will for quite a while, since I think it'll take a while to migrate all of the clients toward `Description`. So I think I want to leave it off for now and discuss what other Boogie-based tool developers would find most workable.

Reply:
How about just a code comment steering folks away from `ErrorData` then?

Reply:
Yep, sounds good!